weights/convert_depth_anything_v2_weights.py

```
@@ -0,0 +1,147 @@
+) -> dict:
+    # Metric checkpoints share the exact same state-dict keys as the relative
+    # ones (the head differs only by a parameter-free Sigmoid * max_depth), so
+    # they cannot be told apart by tensors and would load silently wrong into
+    # the relative ReLU head. Reject by filename - metric depth is out of scope.
+    if "metric" in Path(input_path).name.lower():
+        raise ValueError(
+            f"{input_path} looks like a metric Depth Anything V2 checkpoint. "
+            "LibreYOLO's depth_anything family is relative-depth only; the metric "
+            "head (Sigmoid * max_depth) would load but produce wrong values. "
+            "Metric depth is out of scope."
+        )
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)